code/game/machinery/vending_types.dm

```diff
@@ -790,6 +793,7 @@
 		/obj/item/seeds/chantermycelium = 3,
 		/obj/item/seeds/cherryseed = 3,
 		/obj/item/seeds/chiliseed = 3,
+		/obj/item/seeds/cocaseed = 3,
```

Comment:
the coca plant should probably not be available to the general public, considering it's a narcotic.

Reply:
Will ask lore regarding Biesel law.